xds/src/main/java/io/grpc/xds/XdsComms2.java

```diff
@@ -98,7 +100,26 @@ public void run() {
                       ChannelLogLevel.DEBUG,
                       "Received an EDS response: {0}", clusterLoadAssignment);
                   firstEdsResponseReceived = true;
-                  adsStreamCallback.onEdsResponse(clusterLoadAssignment);
+                  EndpointUpdate.Builder endpointUpdateBuilder = EndpointUpdate.newBuilder();
```

Comment:
I am worried about new code in `XdsComms2` which is going to be replaced by `XdsClientImpl`. Can you put this code in some "helper" file and much later refactor the helper code into the new XdsClientImpl? As I understand you are moving code that was in `LookasideChannelLb.AdsStreamCallbackImpl.onEdsResponse` here.

Reply:
Made a helper function now.